docs/advanced-config.md

```diff
@@ -89,10 +89,35 @@ export OTEL_PROPAGATORS=b3multi
 
 | System property          | Environment variable     | Default value  | Support | Description |
 | ------------------------ | ------------------------ | -------------- | ------- | ----------- |
-| `otel.traces.sampler`    | `OTEL_TRACE_SAMPLER`     | `always_on`    | Stable  | The sampler to use for tracing.	|
+| `otel.traces.sampler`    | `OTEL_TRACES_SAMPLER`     | `always_on`    | Stable  | The sampler to use for tracing.	|
 
-Set `otel.traces.sampler` to `internal_root_off` to drop all traces with root spans where `spanKind` is `INTERNAL`, `CLIENT` or `PRODUCER`. This setting only keeps root spans where `spanKind` is `SERVER` and `CONSUMER`.
+Splunk Distribution of OpenTelemetry Java supports all standard samplers as provided by 
+[OpenTelemetry Java SDK Autoconfigure](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#sampler).
+In addition, we add the following two samplers.
 
+### internal_root_off
+Setting `otel.traces.sampler` to `internal_root_off` will drop all traces with root spans where `spanKind` is `INTERNAL`, `CLIENT` or `PRODUCER`.
+This setting only keeps root spans where `spanKind` is `SERVER` and `CONSUMER`.
+
+### rules
+This sampler allows to ignore individual endpoints and drop all traces that would originate from them.
+For example, the following configuration results in all requests to `/healthcheck` to be excluded from monitoring: 
+
+```shell
+export OTEL_TRACES_SAMPLER=rules
+export OTEL_TRACES_SAMPLER_ARG=drop=/healthcheck;default=parentbased_always_on
```

Comment:
My question is this: is this how otel intends to support complex sampling options? If we document this, and use this format, we're going to be stuck with it, even if otel goes a different way.

Reply:
I don't know of any plans nor opinions of OpenTelemetry for configuring "complex" samplers via environment variable. I don't know if are ready to commit this configuration. We have to give this to our customer who has requested this functionality and ask for their feedback.

Reply:
How would I "drop" more than one endpoint?

Reply:
> How would I "drop" more than one endpoint?

Just add several `drop` rules

Reply:
this should probably be documented here. It definitely wasn't clear to me from reading it.